timescaledb/quick-start/python.md

```diff
@@ -226,73 +221,100 @@ Congratulations, you've successfully inserted data into TimescaleDB using Python
 then add this line to your list of `import` statements:
 
 ```python
 from pgcopy import CopyManager
 ```
 
-Here's some sample code which shows how to insert data into Timescale using `pgcopy`,
-using the example of sensor data from four sensors:
-
-```python
-#insert using pgcopy
-def fast_insert(conn):
-   cur = conn.cursor()
-
-   #for sensors with ids 1-4
-   for id in range(1,4,1):
-       data = (id, )
-       #create random data
-       simulate_query = """SELECT  generate_series(now() - interval '24 hour', now(), interval '5 minute') AS time,
-       %s as sensor_id,
-       random()*100 AS temperature,
-       random() AS cpu
-       """
-       cur.execute(simulate_query, data)
-       values = cur.fetchall()
-
-       #define columns names of the table you're inserting into
-       cols = ('time', 'sensor_id', 'temperature', 'cpu')
-
-       #create copy manager with the target table and insert!
-       mgr = CopyManager(conn, 'sensor_data', cols)
-       mgr.copy(values)
-
-   #commit after all sensor data is inserted
-   #could also commit after each sensor insert is done
-   conn.commit()
-
-   #check if it worked
-   cur.execute("SELECT * FROM sensor_data LIMIT 5;")
-   print(cur.fetchall())
-   cur.close()
-```
-
-
 ### Step 1: Get data to insert into database
 First we generate random sensor data - you would replace this step with funneling
 in your real data from your data pipeline.
 
+```python
+# for sensors with ids 1-4
+for id in range(1, 4, 1):
+    data = (id,)
+    # create random data
+    simulate_query = """SELECT generate_series(now() - interval '24 hour', now(), interval '5 minute') AS time,
+                               %s as sensor_id,
+                               random()*100 AS temperature,
+                               random() AS cpu
+                            """
+    cursor.execute(simulate_query, data)
+    values = cursor.fetchall()
+```
+
 ### Step 2: Define columns of table you're inserting data into
 Then we define the column names of the table we want to insert data into. In this
 case, we're using the `sensor_data` hypertable that we created in the
 "Generate a Hypertable" section above. This hypertable consists of the columns
 named `time`, `sensor_id`, `temperature` and `cpu`. We define these column names
-in a tuple of strings called `cols`.
+in a list of strings called `cols`.
+
+```python
+cols = ['time', 'sensor_id', 'temperature', 'cpu']
+```
 
 ### Step 3: Instantiate a CopyManager with your target table and column definition
 Lastly we create an instance of the `pgcopy` CopyManager, `mgr`, and pass our
-connection variable, hypertable name, and tuple of column names. Then we use
+connection variable, hypertable name, and list of column names. Then we use
 the `copy` function of the CopyManager to insert the data into the database
-performantly using `pgcopy` and then commit when we're done. There is also sample
-code to check if the insert worked.
+performantly using `pgcopy`.
```

Comment:
Nit, but can we use "quickly" or "efficiently" here instead?

Reply:
LOL. Totally missed this one. I don't mind "performant" (even though it's still not a real word and others hate it), but I've never seen this combination!

Reply:
https://github.com/timescale/docs/commit/bddb2c9871386cfb6dc3c2ac0cab4e1a8cd24c26